tests/mutex_unlock_and_sleep/tests/01-run.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+
+def testfunc(child):
+    for i in range(20):
```

Comment:
Maybe 10 is enough ? I tested on hardware (nucleo-l073) and it takes 2 minutes for the test to finish.

Reply:
I was thinking back and forth here, too - however, I didn't want to make it running too short. I suggest we leave it 20 for now, 2 minutes doesn't sound that bad. We still may adapt later on, if needed. Thanks for reviewing and testing!